src/reaper_osara.cpp

```diff
@@ -331,6 +332,30 @@ bool isItemSelected(MediaItem* item) {
 	return *(bool*)GetSetMediaItemInfo(item, "B_UISEL", NULL);
 }
 
+const char* automationModeAsString(int mode) {
+	// this works for track automation mode and global automation override.
+	switch (mode) {
+	case -1:
```

Comment:
The entire block inside this switch statement needs to be indented one more tab. I'm pretty sure you had this right in your first patch... or maybe I missed it?